src/quantuminspire/api.py

```diff
@@ -76,7 +76,7 @@ def __init__(self, base_uri: str, authentication: coreapi.auth.BasicAuthenticati
         except (CoreAPIException, TypeError) as ex:
             raise ApiError('Could not connect to {}'.format(base_uri)) from ex
 
-    def get(self, uri_path: str) -> Any:
+    def _get(self, uri_path: str) -> Any:
```

Comment:
get should be private according to the ticket, i.e., __get not _get.

Reply:
One underscore is considered as a private method/instance variable in python. With two underscores the name is mangled. I don't see why that is necessary. Further, all the other private methods within this class have one underscore (_action, _load_schema, _create_asset etc.)

Reply:
I agree, I think single underscore is enough since that means, in python world, that it should not be accessed outside the class. But in our guidelines it states that __ is private and _ is protected.
But...
To clarify I talked to Koen and he ment it should "not be public" so I'm happy with the single underscore.